src/peergos/server/Main.java

```diff
@@ -358,7 +358,7 @@ public static final void poststrap(Args args) {
             Stream.of(
                     new Command.Arg("port", "service port", false, "9000"),
                     new Command.Arg("peergos-url", "Address of the Peergos server to connect to", false, "http://localhost:8000"),
-                    new Command.Arg("domain-suffix", "Domain suffix to accept", false, ".public.localhost:9000"),
+                    new Command.Arg("domain-suffix", "Domain suffix to accept", false, ".peergos.localhost:9000"),
```

Comment:
the comment for the PR references public and not peergos. The PR comment should be updated.